### ISSUE TITLE
notification bar

### ISSUE THREAD

**Issue by GoogleCodeExporter** [everythingandroid/android-smspopup#196]
```
What steps will reproduce the problem?
1.receiving a text message
2.opening the message
3.closing the message and it still show the message on notification bar

What is the expected output? What do you see instead?
After opening a message the notification bar should clear and show no new 
messages. The notification bar is still showing unread messages when there are 
non.  

What version of the product are you using? On what operating system?

I'm using version 1.0.9 on android 2.2
Please provide any additional information below.

Basically it shows that there are unread messages. To locate the problem I 
disabled my phones notifications and I still happened. Next I disabled sms 
popup and the problem stopped. I uninstalled and re-downloaded but the same 
problem started occurring again.

```

Original issue reported on code.google.com by `moore...@gmail.com` on 21 Oct 2010 at 8:07
